README.md

```diff
@@ -86,5 +94,15 @@ app.after(() => {
   });
 });
 
-await app.ready();
+async function run() {
```

Comment:
why this change?

Reply:
I made that change because the first example in the README, `How to use?`, is compatible to both CommonJS and ES modules projects while the second one, as it uses a top-level await, is only available to ES modules projects.

With this now users can just copy-paste the second `How to use together with @fastify/swagger` example without bothering to adapt it to ES modules (just like the first one).